app/api/helpers/events.py

```diff
@@ -10,42 +10,121 @@ def create_custom_forms_for_attendees(event):
     """
     # common values
     form = 'attendee'
-    is_required = True
-    is_included = True
-    is_fixed = True
     event_id = event.id
     form_type = 'text'
+    is_required = False
+    is_included = False
 
-    first_name_form = CustomForms(
-        form=form,
-        is_required=is_required,
-        is_included=is_included,
-        is_fixed=is_fixed,
-        event_id=event_id,
-        type=form_type,
-        field_identifier='firstname',
-    )
-
-    last_name_form = CustomForms(
-        form=form,
-        is_required=is_required,
-        is_included=is_included,
-        is_fixed=is_fixed,
-        event_id=event_id,
-        type=form_type,
-        field_identifier='lastname',
-    )
-
-    email_form = CustomForms(
-        form=form,
-        is_required=is_required,
-        is_included=is_included,
-        is_fixed=is_fixed,
-        event_id=event_id,
-        type='email',
-        field_identifier='email',
-    )
-
-    save_to_db(first_name_form, 'First name form saved')
-    save_to_db(last_name_form, 'Last name form saved')
-    save_to_db(email_form, 'Email form saved')
+    form_dict = CUSTOM_FORM_IDENTIFIER_NAME_MAP[form]
+    for x in form_dict:
+        form_name = x + "_form"
+        form_type = 'email' if x == 'email' else 'text'
+
+        is_required = True if x in ['firstname', 'lastname', 'email'] else False
+        is_included = (
+            True
+            if x
+            in [
+                'firstname',
+                'lastname',
+                'email',
+                'address',
+                'city',
+                'state',
+                'country',
+                'jobTitle',
+                'phone',
+                'taxBusinessInfo',
+                'company',
+                'website',
+                'twitter',
+                'github',
+            ]
```

Comment:
Add this info in the dict itself

Reply:
@iamareebjamal This custom_form_identifier_name_map dictionary is used in some more places too so that can bring some impact there should I change the code in all the instances where it is used? :sweat_smile:

Reply:
No, just in the dict